tests/h/search/query_test.py

```diff
@@ -513,11 +513,31 @@ def search(self, search):
 
 
 class TestUsersAggregation(object):
+    def test_it_aggregates_annotations_on_users(self, Annotation, search):
+        ann1 = Annotation(userid="acct:pa@example.com")
+        ann2 = Annotation(userid="acct:pa@example.com")
+        ann3 = Annotation(userid="acct:pb@example.com")
 
-    @pytest.fixture
-    def search(self, search):
         search.append_aggregation(query.UsersAggregation())
-        return search
+        result = search.run({})
+
+        count_for_pa = next(r for r in result.aggregations.get('users') if r["user"] == "acct:pa@example.com")['count']
+        count_for_pb = next(r for r in result.aggregations.get('users') if r["user"] == "acct:pb@example.com")['count']
+
+        assert len(result.aggregations.get('users')) is 2
+        assert count_for_pa is 2
+        assert count_for_pb is 1
+
+    def test_it_limits_annotation_aggregation_on_users(self, Annotation, search):
+        for i in range(10):
+            Annotation(userid="acct:pa@example.com")
+
+        search.append_aggregation(query.UsersAggregation(limit=5))
+        result = search.run({})
+
+        count_for_user_a = next(r for r in result.aggregations.get('users') if r["user"] == "acct:pa@example.com")['count']
```

Comment:
Shouldn't this count returned be limited to 5?

Reply:
The `limit` parameter refers to the _number_ of buckets, not the counts for each bucket. See https://hyp.is/KvOyhoNjEeiTa6dg6L6VGw/www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html